### DISCUSSION TITLE
Naming convention for auth-server-url config file for keycloak-js

### DISCUSSION THREAD

**Discussion by rrodriguezOT** [keycloak/keycloak#35735] [Ideas]
Hi,

I found that, in keycloak-js adapter, when an object is passed to the constructor, the object has three properties ```url```, ```realm``` and ```clientId``` but when a string is passed pointing to a json with the object in it, the ```url``` parameter is named ```auth-server-url```.

I forked the repo in case this is a good idea for a change in the javascript adapter, contribute to the project.

Should all the properties be named ```url``` or should be named ```auth-server-url```?

Thank you

**Comment by jonkoops** [comment]
We actually actively recommend against using the JSON file variant (see documentation), so there is no need to contribute a fix, as this functionality will likely be removed at some point in the future.
